src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -151,6 +153,8 @@
 
     private static final String API_COMMIT_STATUS_PATH = "/rest/build-status/1.0/commits{/hash}";
     private static final Integer DEFAULT_PAGE_LIMIT = 200;
+    private static final int TOO_MANY_REQUESTS_HTTP_STATUS = 429;
```

Comment:
Why using `int` instead of `Integer` like above?

Reply:
Because the return type of response.getStatusLine().getStatusCode() against which TOO_MANY_REQUESTS_HTTP_STATUS is compared is a primitive - int.

And we do not need any conversion or manipulation of this value, hence I don't think there's a need for using Integer here.

Reply:
Agree